src/Faker/Provider/pt_BR/PhoneNumber.php

```diff
@@ -33,14 +33,13 @@ public static function areaCode()
      * Generates a 8/9-digit cellphone number without formatting characters.
      * @param bool $formatted [def: true] If it should return a formatted number or not.
      * @param bool $ninth     [def: false] If the number should have a nine in the beginning or not.
-     *                        If the generated number begins with 7 this is ignored.
```

Comment:
It's explained in the docs for `$ninthDigitAreaCodes`. In short, Brazilian cellphones are receiving a ninth digit in a phased way, across our area codes.